### PR TITLE
Reset cookbooks and benchmarks paths

### PR DESCRIPTION
**Pull request by cmills1095**
Set up shortcuts and restructured doc/sphinx/user such that all cookbooks and benchmarks files are in aspect/cookbooks and aspect/benchmarks

Pull Request Checklist. Please read and check each box with an X. Delete any part not applicable. Ask on the [forum](https://community.geodynamics.org/c/aspect) if you need help with any step.

*Describe what you did in this PR and why you did it.*

### Before your first pull request:

* [ ] I have read the guidelines in our [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md) document.

### For all pull requests:

* [ ] I have followed the [instructions for indenting my code](../blob/main/CONTRIBUTING.md#making-aspect-better).

### For new features/models or changes of existing features:

* [ ] I have tested my new feature locally to ensure it is correct.
* [ ] I have [created a testcase](http://www.math.clemson.edu/~heister/manual.pdf#sec%3Awriting_tests) for the new feature/benchmark in the [tests/](../blob/main/tests/) directory.
* [ ] I have added a changelog entry in the [doc/modules/changes](../blob/main/doc/modules/changes) directory that will inform other users of my change.

### CODE REVIEW

File: doc/sphinx/user/benchmarks/benchmarks
Comment (tjhei) [inline doc/sphinx/user/benchmarks/benchmarks:1]: This seems incorrect.

**Review by tjhei — APPROVED** [review]

File: doc/sphinx/user/benchmarks/benchmarks
Comment (cmills1095) [reply doc/sphinx/user/benchmarks/benchmarks:1]: Thanks for the catch, we were stumped. I set up the symlink wrong, we're trying it again

**Review by gassmoeller — APPROVED** [review]
This should now be ready. It can be merged when the testers are done, and afterwards we can start fixing up the cookbooks.

Note for future: Linux symlinks work in git repositories and sphinx documentation, but windows shortcuts do not (because they contain absolute paths, not relative paths).

**Comment by bangerth** [comment]
Ah, fantastic -- Windows in 2022 does not have a feature that's been part of Unix since 1978... From wikipedia:

> Symbolic links were already present by 1978 in minicomputer operating systems from [DEC](https://en.wikipedia.org/wiki/Digital_Equipment_Corporation) and Data General's [RDOS](https://en.wikipedia.org/wiki/Data_General_RDOS). Today they are supported by the [POSIX](https://en.wikipedia.org/wiki/POSIX) operating system standard, most [Unix-like](https://en.wikipedia.org/wiki/Unix-like) [operating systems](https://en.wikipedia.org/wiki/Operating_system) such as [FreeBSD](https://en.wikipedia.org/wiki/FreeBSD), [Linux](https://en.wikipedia.org/wiki/Linux), and [macOS](https://en.wikipedia.org/wiki/MacOS). Limited support also exists in Windows operating systems such as [Windows Vista](https://en.wikipedia.org/wiki/Windows_Vista), [Windows 7](https://en.wikipedia.org/wiki/Windows_7) and to some degree in [Windows 2000](https://en.wikipedia.org/wiki/Windows_2000) and [Windows XP](https://en.wikipedia.org/wiki/Windows_XP) in the form of shortcut files.

**Comment by gassmoeller** [comment]
We forgot to give this the ready to test label, but the duplicate github tester succeeded, so I am going to merge this (this PR does not affect anything but the documentation anyway).

**Comment by gassmoeller** [comment]
Actually I cannot merge it, the required tester is not done yet. Whenever the tester is done please go ahead and merge so we can get started on the cookbooks.

**Comment by naliboff** [comment]
@cmills1095 @gassmoeller - FYI, it looks there are quite a few merge conflicts. I'm not sure what this is due to (other fixes from recent PRs), but I wanted to let you know in case you had not seen this yet.

**Comment by gassmoeller** [comment]
I will take care of it, I should have merged this this morning, but had no time.

**Comment by gassmoeller** [comment]
Alright, next try. All required tests have passed, the only remaining one is a duplicate. To prevent further conflicts I am merging this now.

**Pull request merged**
